sqlalchemy_ibmi/base.py

```diff
@@ -802,10 +802,6 @@ def _get_default_schema_name(self, connection):
             default_schema_name = default_schema_name.strip()
         return self.normalize_name(default_schema_name)
 
-    @property
-    def default_schema_name(self):
-        return self._get_default_schema_name()
```

Comment:
I didn't think removing this would work, but it looks like the DefaultDialect will call `_get_default_schema_name` during `initialize`: https://github.com/zzzeek/sqlalchemy/blob/master/lib/sqlalchemy/engine/default.py#L311-L314 Awesome, though it probably should have been done in a separate PR to explain why.